addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,796 @@
-                        <h1>No event found</h1>
-                        <p groups="event.group_event_manager">Use the top menu <i>'Content'</i> to create your first event.</p>
-                    </div>
-                    <div t-foreach="event_ids" t-as="event" class="card mb-3" itemscope="itemscope" itemtype="http://schema.org/Event">
-                        <div t-attf-class="card-body #{event.is_participating and 'o_wevent_participating_container' or ''}">
-                            <span t-if="event.is_participating" class="badge badge-info">Participating</span>
-                            <span t-if="event.is_online" class="badge badge-info">Online</span>
-                            <span t-if="not event.website_published" class="badge badge-danger">unpublished</span>
-                            <small class="badge badge-primary float-right" t-if="event.event_type_id">
-                                <i class="fa fa-ticket" role="img" aria-label="Event type" title="Event type"/> <span t-field="event.event_type_id"/>
-                            </small>
-                            <h4>
-                                <a itemprop="url" t-att-class="event.state == 'done' and 'text-success'" t-attf-href="/event/#{ slug(event) }/#{(not event.menu_id) and 'register' or ''}"><b itemprop="name" t-field="event.name"/></a>
-                            </h4>
-                            <small t-if="event.organizer_id" class="text-muted">Organized by <span t-field="event.organizer_id" /></small>
-                            <div>
-                                <i class="fa fa-clock-o text-primary mx-1" role="img" aria-label="Start date" title="Start date"/>
-                                <span itemprop="startDate" t-field="event.with_context(tz=event.date_tz).date_begin" t-options='{"hide_seconds":"True"}'></span>
-                                <small>to</small>
-                                <span itemprop="endDate" t-field="event.with_context(tz=event.date_tz).date_end" t-options='{"hide_seconds":"True"}'></span>
+        <div id="wrap" class="o_we_index">
+            <!-- Options -->
+            <t t-set="opt_events_list_cards" t-value="request.website.viewref('website_event.opt_events_list_cards').active"/>
+            <t t-set="opt_events_list_columns" t-value="request.website.viewref('website_event.opt_events_list_columns').active"/>
+            <!-- Topbar -->
+            <t t-call="website_event.index_topbar"/>
+            <!-- Drag/Drop Area -->
+            <div id="oe_structure_we_index_1" class="oe_structure"/>
+            <!-- Content -->
+            <main t-attf-class="o_we_events_list #{'opt_event_list_cards_bg' if opt_events_list_cards else ''}">
+                <div class="container">
+                    <div class="row">
+                        <div id="o_we_index_main_col" t-attf-class="col-md my-5 #{'opt_events_list_columns' if opt_events_list_columns else 'opt_events_list_rows'}">
```

Comment:
Mmh I see you used `o_we` prefix for your classes... use `o_wevent`, it's clearer and we used `o_we` for web_editor... :/

Also, important: change all your `A if hello else B` stuff in `hello and A or B`. This is especially useful when B is `''` since in that case you can write `hello and A` which is a huge difference because: `t-att-class="'hey' if False else ''"` will generate an empty `class=""` attribute in the final DOM while `t-att-class="False and 'hey'"` will not.

Reply:
Let's flip a coin. If you loose, you change the prefix of the web_editor.
Heads, you loose. Tails, I win.

Reply:
web_editor is already a merged task, you loose :)